include/itkPhaseCorrelationOperator.h

```diff
@@ -69,8 +66,31 @@ class ITK_TEMPLATE_EXPORT PhaseCorrelationOperator :
   itkTypeMacro(PhaseCorrelationOperator, ImageToImageFilter);
 
   /** ImageDimension enumeration. */
-  itkStaticConstMacro(ImageDimension, unsigned int,
-                      VImageDimension);
+  itkStaticConstMacro(ImageDimension, unsigned int, VImageDimension);
+
+  /** Set/Get control points for band-pass filtering.
+   * These points represent radial distances from constant component.
+   * The values of these points should be specified on 0.0-1.0 scale,
+   * 0.0 relating to constant frequency component, and 1.0 relating 
+   * to highest frequency along all dimensions.
+   * Point values should be monotonitcally increasing:
+   * 0 <= p[0] < p[1] < p[2] < p[3] <= 1.0
+   * All frequencies below p[0] and above p[3] are completely removed.
+   * All frequencies between p[1] and p[2] are completely kept.
+   * Frequencies between p[0] and p[1]; and between p[2] and p[3]
+   * are transition zones with linearly increasing (p0-p1)
+   * and decreasing (p2-p3) coefficient of influence. Function plot:
+   *   ^
+   *  1|             _________
+   *   |            /         \
+   *   |           /           \
+   *   |          /             \
+   *  0|    *----*---*-------*---*----*
+   * coef  0.0   p0  p1      p2  p3  1.0
+   */
```

Comment:
Use a Butterworth filter here to prevent phase distortion.

Reply:
This is still planned to be done, just not in this PR.